samples/highcharts/accessibility/accessible-dynamic-data-animating/demo.js

```diff
@@ -0,0 +1,68 @@
+const chart = Highcharts.chart('container', {
+    title: {
+        text: 'Dynamic data'
+    },
+    subtitle: {
+        text: 'Click button to animate or explore chart'
+    },
+    accessibility: {
+        enabled: false
+    },
+    tooltip: {
+        dateTimeLabelFormats: {
+            day: '%H:%M',
+            hour: '%H:%M'
+        }
+    },
+    xAxis: {
+        type: 'datetime',
+        dateTimeLabelFormats: {
+            day: '%H:%M',
+            hour: '%H:%M'
+        }
+    },
+    plotOptions: {
+        series: {
+            pointStart: 0,
+            pointInterval: 1000 * 60 * 60
+        }
+    },
+    series: [{
+        name: 'Random data',
+        data: [1, 3, 4, 6, 7, 5, 3, 4, 8, 9, 7, 6, 4, 3]
+    }]
+});
+
+let intervalId;
+let isAnimating = false;
+
+const toggleButton = document.getElementById('toggle');
+toggleButton.onclick = function () {
+    toggleAnimation();
+};
```

Comment:
Unnecessary nesting. May have to move this below the function definition though.

```suggestion
toggleButton.onclick = toggleAnimation;
```